HIC.Extensions.nuspec

```diff
@@ -10,10 +10,10 @@
 		</dependencies>
     </metadata>
   <files>
-    <file src="Plugin\windows\bin\$configuration$\net6.0-windows\win-x64\publish\*" 

```

Comment:
We can actually drop this whole section, since we aren't using the `nuspec` tool to package the plugin any more, it doesn't need to know the file locations!